client/src/pages/SignUp/SignUp.tsx

```diff
@@ -42,18 +43,26 @@ export default function Register(): JSX.Element {
       <CssBaseline />
       <Grid item xs={12} sm={8} md={7} elevation={6} component={Paper} square>
         <Box className={classes.authWrapper}>
-          <AuthHeader linkTo="/login" asideText="Already have an account?" btnText="Login" />
+          {/* --Commented out until thew new authHeader is completed
+          <AuthHeader linkTo="/signup" asideText="Don't have an account?" btnText="Create account" />*/}
```

Comment:
Same here with regards to react-router-dom `Link` component

Reply:
I have un-commented all code, as requested!